pkg/placement/membership.go

```diff
@@ -226,23 +225,10 @@ func (p *Service) performTableDissemination(ctx context.Context, ns string) erro
 		streams = append(streams, *stream)
 	})
 
-	// Check if the cluster has daprd hosts that expect vnodes;
-	// older daprd versions (pre 1.13) do expect them, and newer versions (1.13+) do not.
 	req := &tablesUpdateRequest{
 		hosts: streams,
 	}
-	// Loop through all streams and check what kind of tables to disseminate (with/without vnodes)
```

Comment:
We don't need to disseminate tables with vnodes anymore, so we're removing this.